sql/report_queries/acrl/collections_expenditures/ACRL Collection Expenditures A.sql

```diff
@@ -0,0 +1,207 @@
+--This query is for materials purchased, not other cost ACRL suggest may be included under "All other materials/services cost"
+--This query provides a total amount of material expenditures, broken down by invoice lines and including any adjustment made at the invoice level.
+--An export to Excel will provide the total amount paid by summing the 'total paid converted' column
```

Comment:
I'm not sure I understand how this is worded. I originally took it to mean that to get the sum will be done in Excel once I exported the results. With the community edition, I exported as a csv but have the same results as in DBeaver. Do you mean that you take all the results in this column and sum them up? Sorry to be so dense. Could you clarify this here? Thanks

Reply:
Yes, you will need to export your results and sum the 'total paid converted column'.